### PR TITLE
Image & input

### PR DESCRIPTION
**Pull request by Amandeep-sinha**
Added Image and Input component

check live at :- 
https://hardcore-aryabhata-201d65.netlify.app/image

### CODE REVIEW

**Review by Varu98 — APPROVED** [review]

File: Image.css [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    margin:1rem 0rem;
+    border-radius: 70%;
+    max-width:10rem;
+    height:10rem ;
+}
+.round{
+    width:40%;
+    
+}
+#round-border{
```
Comment (msdeshmukh009) [inline Image.css:32]: Maybe avoid Id selector for styling.

File: Input.css [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+}
+.active{
+    width:40vw;
+    height:fit-content;
+    background-color: white;
+    border-radius: 5px;
+    padding:2rem;
+    margin:3rem;
+}
+form{
```
Comment (msdeshmukh009) [inline Input.css:13]: Styling an element directly is not a good idea. As the user might not want the same style for every element.

**Review by msdeshmukh009 — CHANGES_REQUESTED** [review]
Consider making the following changes. You are using id selectors in a lot of places which is not really great practice. And also styling the elements directly changes the meaning of that element so maybe refactor that as well.

File: Input.css (hunk 2 above)
Comment (Amandeep-sinha) [reply Input.css:13]: okay mahesh from next time will keep in mind

**Pull request merged**